openapiGenerator.go

```diff
@@ -121,6 +122,10 @@ type openapiGenerator struct {
 	// If set to true, kubebuilder markers and validations such as PreserveUnknownFields, MinItems, default, and all CEL rules will be omitted from the OpenAPI schema.
 	// The Type and Required markers will be maintained.
 	disableKubeMarkers bool
+
+	// If provided, ignores the contained kubebuilder markers and validations, and prevents them from being applied to the OpenAPI schema.
+	// If none are provided, no markers will be ignored.
+	ignoredKubeMarkers []string
```

Comment:
Could you note the syntax for the markers that are ignored - regex, subtring, exact, etc,?